themes/bootstrap3/templates/record/illrequest.phtml

```diff
@@ -55,7 +55,11 @@
 
   <?php if (in_array("pickUpLibraryLocation", $this->extraFields)): ?>
     <div class="form-group">
-      <label id="pickupLibraryLocationLabel" class="control-label"><i></i>&nbsp;<?=$this->transEsc("ill_request_pick_up_location")?>:<noscript> (<?=$this->transEsc("Please enable JavaScript.")?>)</noscript></label>
+      <label id="pickupLibraryLocationLabel" class="control-label">
+        <span class="loading-icon"><?=$this->icon('spinner', 'icon--spin') ?></span>
```

Comment:
Is the icon--spin class needed here? It's not used in other instances of the spinner icon.

Reply:
I tested that this is not necessary following commit 3543908, so I have removed it.